apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -117,7 +117,7 @@ var NetSimPacketEditor = module.exports = function (initialConfig) {
    * @type {Number}
    * @private
    */
-  this.maxPacketSize_ = initialConfig.maxPacketSize || Infinity;
+  this.maxPacketSize_ = initialConfig.maxPacketSize || 8192;
```

Comment:
On that note, the comment on `this.maxPacketSize_` should probably call out that it's measured in bits.